sparse/coo/core.py

```diff
@@ -1436,6 +1436,22 @@ def __rmatmul__(self, other):
         except NotImplementedError:
             return NotImplemented
 
+    def __array_function__(self, func, types, args, kwargs):
+        import sparse as module
+        for submodule in func.__module__.split('.')[1:]:
```

Comment:
```suggestion
        try:
            submodules = func.__module__.split('.')[1:]
            for submodule in submodules:
                module = getattr(module, submodule)
            sparse_func = getattr(module, func.__name__)
            return sparse_func(*args, **kwargs)
        except AttributeError:
            pass
        if not hasattr(type(self), func.__name__):
            return NotImplemented
            
        sparse_func = getattr(type(self), func.__name__)
        return sparse_func(*args, **kwargs)
```

Reply:
Oh, I see now. Indeed, that's a great solution, committing!

Reply:
Yes, GitHub has no way to do multiline suggestions AFAICT.